graylog2-server/src/main/java/org/graylog2/buffers/processors/OutputBufferProcessor.java

```diff
@@ -52,6 +53,9 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(OutputBufferProcessor.class);
 
+    public static final String INCOMING_MESSAGES_METRICNAME = name(OutputBufferProcessor.class, "incomingMessages");
+    public static final String PROCESS_TIME_METRICNAME = name(OutputBufferProcessor.class, "processTime");
```

Comment:
Left from before I introduced the GlobalMetricNames class.
Will reduce visibility.